tasks/main.yml

```diff
@@ -10,6 +10,12 @@
   user:
     name: verdaccio
 
+- name: resolve npm config path
+  command: npm config get prefix 
```

Comment:
Could you ensure that only the first line gets return by using e.g. head -1? On my machine the command returned:

```
/usr/local/lib/npm


   ╭────────────────────────────────────────────────────────────────╮
   │                                                                │
   │      New patch version of npm available! 6.14.6 → 6.14.7       │
   │   Changelog: https://github.com/npm/cli/releases/tag/v6.14.7   │
   │               Run npm install -g npm to update!                │
   │                                                                │
   ╰────────────────────────────────────────────────────────────────╯
```